simulator/setting_up_simulator.md

```diff
@@ -6,19 +6,33 @@ title: Setting up the Simulator
 Setting up the Simulator
 ========================
 
-## Prerequisites
+There are three key components you will need in order to be able to use the simulator:
 
+- [Webots](#webots), the platform which runs the simulation
+- [Python](#python), the interpreter for the your robot code
+- The [world simulation](#world-simulation), which includes the SR API for use in the simulator
 
-### Webots
 
-Our simulator runs in a program called Webots.
-In order to run the simulation you first need to install Webots.
+## Webots
+
+Webots is a free open-source robot simulator and is the program which runs the simulation.
+
 You need to download [Webots from here](https://cyberbotics.com/#download) and install it.
 
 The current supported version is "R2023b".
 
+### Troubleshooting: performance
+
+The default settings are fine for most users however if you are using a less powerful computer or one without a dedicated graphics card (as is the case on many laptops), you may wish to adjust the graphics settings to enable the simulation to run faster.
+
+If you find that the simulation runs very slowly we suggest disabling both Ambient Occlusion and Shadows.
+These should not affect the behaviour of the simulation, only the rendered visuals.
 
-### Python
+To do this on Windows, open webots and go to the menu **Tools** &rarr; **Preferences** &rarr; **OpenGL**, then set **Ambient Occlusion** to "none" and check the box next to "Disable shadows".
+
+To do this on Mac, open webots and go to the menu **Webots** &rarr; **Preferences** &rarr; **OpenGL**, then set **Ambient Occlusion** to "none" and check the box next to "Disable shadows".
```

Comment:
Note: that this is the right place to make this change is a guess. Please could someone with a mac check this?

Reply:
Yes. 
<img width="330" alt="image" src="https://github.com/srobo/docs/assets/6197148/a61c8756-da45-4af9-af96-1316737b2a3e">